openapi/components/schemas/GatewayAccountConfig/PostFinance.yaml

```
@@ -47,3 +47,6 @@ allOf:
           skipPaymentFileUpload:
             type: boolean
             default: false
+          allowAliasAuthentication:
```

```suggestion
          enableAliasAuthentication:
```

To be consistent with 
https://github.com/Rebilly/api-definitions/blob/c9bd3ea56dc2d35e7976c469379d502e2d7a2c7d/openapi/components/schemas/GatewayAccountConfig/PayPal.yaml#L18
https://github.com/Rebilly/api-definitions/blob/c9bd3ea56dc2d35e7976c469379d502e2d7a2c7d/openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml#L47